procurement.md

```diff
@@ -0,0 +1,695 @@
+Doing any of these in an ad-hoc manner will usually raise M&O
+costs.
+
+Sometimes government agencies have recourse to centralized IT
+departments that operate deployments across many agencies and
+departments.  Central management, though, comes with its own
+constraints.  Our research suggests that often the only way to
+effectively manage an entire jurisdictions's worth of technology in
+one central place is to force all that technology to use the same
+technology stacks and to deploy in uniform configurations.  This
+allows centralized IT to fit deploys into standard processes,
+automated tools, and low-knowledge management structures.  This is an
+effective way to lower M&O costs, and a number of jurisdictions have
+adopted this approach.
+
+Of course, there is a cost to restricting agencies to a set of uniform
+technologies arranged in pre-approved configurations.  Central
+agencies who have to consider the implications of technology changes
+jurisdiction-wide sometimes find it difficult to explore new avenues
+as technology evolves.  It is difficult to approve solutions that are
+well-suited to the needs of just one deployment scenario but require
+deviation from the uniform deployment standards.  Every such
+deployment undercuts part of the rationale for centralization.
+
+Regardless of where the deployment eventually resides, though, it is
+reasonable to build a monthly carrying cost into the budget.  Even if
+centralized IT hosts the deploy, it is possible (perhaps even likely)
+that an agency will have to be a motive force behind non-feature
+upgrades for stability and security.
+
+Custom software should not be treated like a depreciable asset that
+gets fixed when broken and replaced when outdated.  Modular design and
+contracting approaches enable an agency to make continuous,
+small-scale investments in the software that can extend its lifespan
+indefinitely.  Even major upgrades and transformations can be
+performed in stages.  The same software development process that
+performs ongoing updates should also tend to maintenance of those
+modules.  For modules that are not currently undergoing active
+development, combine the maintenance with slowly tending to structure
+and paying down the "technical debt" that accumulates in projects and
+degrades them over time.
+
+KEY RECOMMENDATION: hile operation might be a separate line-item,
+  much of the maintenance work should be part of the ongoing, modular
+  improvement that keeps software current and prevents an agency from
+  having to start over.
+
+## Open Source Solicitation
+
+Modular contracting often can be done in small enough contracts that
+streamlined procurement processes are available. At some point,
+though, an agency may want to embark on a larger project that does not
+easily fit into a "small contracting" budgeting provision.  In that
+case, it will need to operate an RFP process.  This section contains
```

Comment:
spell out RFP the first time it is used

Reply:
...picking up on an earlier comment: for example, here, should there be a separate PR for spelling out "RFP" the first time it is used, and a different one for "RFI" below?  I think these fixes should just go directly on `main`.  (But it's technically blocked right now, as far as I can tell.)